provider/aws/formation/g1/app.json.tmpl

```diff
@@ -328,6 +328,15 @@
                 "TargetGroupArn": { "Fn::If": [ "Balancer{{ upper $e.Name }}ALB",
                   { "Ref": "{{ $manifest.BalancerResourceName $e.Name }}TargetGroup" },
                   { "Ref": "AWS::NoValue" }
+                ] },
+                "AccessLoggingPolicy": { "Fn::If": [ "BlankLogBucket",
+                  { "Ref": "AWS::NoValue" },
+                  {
+                    "EmitInterval": 5,
+                    "Enabled": true,
+                    "S3BucketName": { "Ref": "LogBucket" },
+                    "S3BucketPrefix": { "Fn::Sub": "convox/logs/${AWS::StackName}/elb/" }
```

Comment:
Not sure if we should be more specific with the prefix as there could be multiple ELBs for a single app?